pkg/jwtauth/jwtauth.go

```diff
@@ -0,0 +1,67 @@
+/****************************************************************************
+ * Copyright 2020, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+// Package jwtauth contains JWT-related helpers
+package jwtauth
```

Comment:
can we have this as `jwt`?

Reply:
we use jwt package already (from dgrijalva), that's why I think we should have a different name here if possible.

Reply:
Initially I named it this way because It also has `MatchClientSecret` which isn't _really_ JWT-related, but on second thought, I guess it's fine to use `jwt`.